region/norfair/east/Speed Booster Hall.json

```diff
@@ -994,6 +1083,56 @@
         "To speed it up, it helps to prevent breaking most of the crumbles (no more than two consecutive) so the Geruta doesn't get stuck."
       ]
     },
+    {
+      "link": [2, 2],
+      "name": "R-Mode Spark Interrupt, Return through Lava Trap",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_heatedCrystalFlashForReserveEnergy",
+        {"simpleHeatFrames": 310},
+        {"heatFrames": 40},
+        "h_shinechargeMaxRunway",
+        {"shinespark": {"frames": 49, "excessFrames": 10}},
+        "h_shinechargeMaxRunway",
+        "h_heatTriggerRModeSparkInterrupt",
+        {"or": [
+          {"and": [
+            "f_ClearedSpeedBoosterLavaTrap",
+            {"heatFrames": 1250},
+            {"lavaFrames": 40}
+          ]},
+          {"and": [
+            "h_heatedCrystalFlash",
+            {"heatFrames": 2060},
+            {"lavaFrames": 2060}
+          ]},
+          {"and": [
+            "Gravity",
+            "h_heatedCrystalFlash",
+            {"heatFrames": 1250},
+            {"lavaFrames": 1250}
+          ]},
+          {"and": [
+            "Gravity",
+            "h_heatProof",
+            "h_lavaProof"
+          ]}
+        ]}
+      ],
+      "exitCondition": {"leaveNormally": {}},
+      "unlocksDoors": [
+        {"nodeId": 2, "types": ["ammo"], "requires": []}
+      ],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Outrun the rising lava up to the highest ledge and shinecharge before touching the crumble blocks.",
```

Comment:
1) Have you investigated CF at [2] -> wait for health to lower -> shinespark interrupt while still close to 2, under the spikes somewhere?
2) How much is 2000 lava and heat frames of damage?

Reply:
> 1. Have you investigated CF at [2] -> wait for health to lower -> shinespark interrupt while still close to 2, under the spikes somewhere?

Added this strat, requiring the lava trap be already cleared.